README.md

```diff
@@ -40,14 +40,14 @@ Integrating components into your project depends heavily on your setup. The reco
 Within an npm project, you should install the package and save it as a dependency:
 
 ```
-npm install --save office-ui-fabric-react
+npm i office-ui-fabric-react
 ```
 
-This will add the fabric-react project as a dependency in your package.json file, and will drop the project under node_modules/office-ui-fabric-react.
+This will add the fabric-react project as a dependency in your `package.json` file, and will drop the project under `node_modules/office-ui-fabric-react`.
 
 The library includes commonjs entry points under the lib folder. To use a control, you should be able to import it and use it in your render method:
 
-```js
+```javascript

```

Comment:
Curious of the reasoning for this and the ts/typescript change later? Markdown parsers ought to understand either as far as I'm aware.

Reply:
Thank @ecraig12345 about code highlight `js`. I read here https://guides.github.com/features/mastering-markdown/ about code hightlight, I just see `javascript`, then I am not sure about `js`, and it is also follow a popular or easy-to-understand way.